File: Latihan B/JAVA/Processor.java
public class Processor {
    private String name;
    private int price;

    // konsturktor
    public Processor(){}
    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    // set untuk semua variable
    public void setName(String name) {
        this.name = name;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    // get untuk semua variable
    public String getName() {
        return name;
    }
    public int getPrice() {
        return price;
    }

    // tampil untuk processor
    public void printProc() {
        System.out.println("  Nama Processor     : " + name);
        System.out.println("  Harga Processor    : " + price);
    }
}

File: Latihan B/PHP/Processor.php
<?php
class Processor {
    private $name;
    private $price;

    // konsturktor
    function __construct($name, $price) {
        $this->name = $name;
        $this->price = $price;
    }

    //set untuk semua variable
    function setName($name) {
        $this->name = $name;
    }
    function setPrice($price) {
        $this->price = $price;
    }
    //get untuk semua varible
    function getName() {
        return $this->name;
    }
    function getPrice() {
        return $this->price;
    }

    // tampil untuk prosessor
    function printProc() {

        echo "Nama Processor : ".  $this->getName(). "<br/>";
        echo "Harga Processor : ". "Rp. ". $this->getPrice(). "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>